clients/fides-js/src/lib/preferences.ts

```diff
@@ -46,36 +45,26 @@ export const updateConsentPreferences = async ({
   debug?: boolean;
   servedNotices?: Array<LastServedNoticeSchema> | null;
   tcf?: TcfSavePreferences;
-  updateCookie?: (oldCookie: FidesCookie) => Promise<FidesCookie>;
+  updateCookie: (oldCookie: FidesCookie) => Promise<FidesCookie>;
 }) => {
-  // Derive the CookieKeyConsent object from privacy notices
-  const noticeMap = new Map<string, boolean>(
-    consentPreferencesToSave.map(({ notice, consentPreference }) => [
-      notice.notice_key,
-      transformUserPreferenceToBoolean(consentPreference),
-    ])
-  );
-  const consentCookieKey: CookieKeyConsent = Object.fromEntries(noticeMap);
-
   // Derive the Fides user preferences array from privacy notices
-  const fidesUserPreferences: Array<ConsentOptionCreate> =
-    consentPreferencesToSave.map(({ notice, consentPreference }) => {
-      const servedNotice = servedNotices
-        ? servedNotices.find(
-            (n) =>
-              n.privacy_notice_history.id === notice.privacy_notice_history_id
-          )
-        : undefined;
-      return {
-        privacy_notice_history_id: notice.privacy_notice_history_id,
-        preference: consentPreference,
-        served_notice_history_id: servedNotice?.served_notice_history_id,
-      };
-    });
-
-  // Update the cookie object
-  // eslint-disable-next-line no-param-reassign
-  cookie.consent = consentCookieKey;
+  const fidesUserPreferences: Array<ConsentOptionCreate> | undefined =
```

Comment:
this is the same, but wrapped in a ternary to be a little more careful about what preferences we pass to the backend. we don't want to accidentally overwrite notice preferences when working with tcf data!